.nvmrc

```diff
@@ -0,0 +1 @@
+v12.18.4
```

Comment:
This is the way to specify nvm which node version to use, instead of specifying it manually
You could do just nvm install in this repo and it would look up for this node version